File: Rails_full_version/app/controllers/pages_controller.rb
class PagesController < ApplicationController

  def blank_page
       render :layout => "application"
  end

  def profile
      render :layout => "application"
  end

  def signon
      render :layout => "pages"
  end

  def signup
      render :layout => "pages"
  end


  def locked_screen
      render :layout => "pages"
  end


  def page404
      render :layout => "pages"
  end

  def page500
      render :layout => "pages"
  end


end

File: MVC5_full_version/MVC5_full_version/Controllers/PagesController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace MVC5_full_version.Controllers
{
    public class PagesController : Controller
    {
        // GET: Pages
        public ActionResult BlankPage()
        {
            return View();
        }
        public ActionResult Profile()
        {
            return View();
        }
        public ActionResult SignIn()
        {
            return View();
        }
        public ActionResult SignUp()
        {
            return View();
        }
        public ActionResult LockedScreen()
        {
            return View();
        }
        public ActionResult Error404()
        {
            return View();
        }
        public ActionResult Error500()
        {
            return View();
        }
    }
}